backend/app/utils/banService.ts

```diff
@@ -0,0 +1,41 @@
+import BanDAO from '../daos/banDAO'
+import Account from '../entities/account'
+import Ban from '../entities/ban'
+
+class BanService {
+    private banDAO: BanDAO
+
+    constructor() {
+        this.banDAO = new BanDAO()
+    }
+
+    async checkIfUserIsBanned(account: Account) {
+        const ban = await this.banDAO.getBanByUser(account)
+
+        if (ban && ban.banned_to > Date.now()) {
+            const bannedDateobject = new Date(ban.banned_to * 1)
+            const date = bannedDateobject.getDate() + '-' +
+                            bannedDateobject.getMonth() + '-' +
+                            bannedDateobject.getFullYear() + ' ' +
+                            bannedDateobject.getHours() + ':' +
+                            bannedDateobject.getMinutes()
```

Comment:
```suggestion
            const date = `
                ${ bannedDateobject.getDate() }-
                ${ bannedDateobject.getMonth() }-
                ${ bannedDateobject.getFullYear() } 
                ${ bannedDateobject.getHours() }:
                ${ bannedDateobject.getMinutes()}
```

Something like this works too I think. Don't just merge, please test first.

Reply:
Het werkte niet op deze manier en heb het een soortvan toegepast maar ik denk niet dat het beter is